models/item.js

```diff
@@ -1,29 +1,72 @@
 'use strict';
-
 const VALID_PARAMETERS = [ 'completed', 'title', 'description' ]
 
 module.exports = function(sequelize, DataTypes) {
-  var Item = sequelize.define('Item', {
-    title: DataTypes.STRING,
-    description: DataTypes.TEXT,
-    completed: DataTypes.BOOLEAN,
-    parent_id: DataTypes.INTEGER,
-    user_id: DataTypes.INTEGER
-  }, {
-    classMethods: {
-      associate: function(models) {
-        // associations can be defined here
+  const Item = sequelize.define('Item',
+    {
+      title: DataTypes.STRING,
+      description: DataTypes.TEXT,
+      completed: DataTypes.BOOLEAN,
+      parent_id: DataTypes.INTEGER,
+      user_id: DataTypes.INTEGER
+    },
+
+
+
+    {
+      hooks: {
+        // afterCreate: function(item, options) {
+        // },
+        // afterDestroy: function(user, options) {
+        //   user.username = 'Toni'
+        // },
+        afterUpdate: function(item, options) {
+          const Audit = sequelize.import('../models/audit');
```

Comment:
We should import this at the top of this module, so that it need not be re-imported for every execution of this hook.